processor/processor.go

```diff
@@ -2589,3 +2579,13 @@ func (proc *HandleT) updateRudderSourcesStats(ctx context.Context, tx jobsdb.Sto
 	err := rsourcesStats.Publish(ctx, tx.SqlTx())
 	return err
 }
+
+func filterConfig(eventCopy *transformer.TransformerEventT, destination *backendconfig.DestinationT) {
+	if configsToFilterI, ok := destination.DestinationDefinition.Config["configFilters"]; ok {
+		if configsToFilter, ok := configsToFilterI.([]string); ok {
```

Comment:
In practice, casting to `[]string` has no chances for success, since this config will be populated through json unmarshalling. 
https://go.dev/play/p/BLqiaLRfgO-

- Please adapt relevant tests to use proper, unmarshalled configs
- Make sure to typecheck the next casting that will be introduced too (`k, ok := configKey.(string); ok`) as a slice of `any` type can literally hold **any type**

Reply:
Got it addressed the same, added unit test cases as well